src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden) {
+    super(...arguments);
+
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
```

Comment:
`Herbivore has a method of hide, which changes the hidden property of the beast to the opposite value and helps to hide from carnivores.`
```suggestion
  hide() {
    this.hidden = !this.hidden;
  }
```